weave-js/src/components/Dialog/Dialog.tsx

```diff
@@ -51,11 +52,12 @@ export const Overlay = React.forwardRef(
  */
 const contentClassName = classNames(
   'night-aware',
-  'absolute left-[50%] top-[50%] translate-x-[-50%] translate-y-[-50%]', // centers modal on screen
+  'fixed left-[50%] top-[50%] translate-x-[-50%] translate-y-[-50%]', // centers modal on screen
```

Comment:
`absolute` positions in relation to `body` element, meaning the dialog will be displayed at the top of the page, potentially out of view on pages that scroll.

updating to `fixed` ensures it's centered on the actual view port, regardless of where user is scrolled to on the page